aws-lc-sys/builder/cmake_builder.rs

```diff
@@ -298,10 +311,14 @@ impl crate::Builder for CmakeBuilder {
         let mut missing_dependency = false;
 
         if target_os() == "windows" {
-            if target_arch() == "x86_64" && !test_nasm_command() && !is_no_asm() {
+            if target_arch() == "x86_64"
+                && !is_no_asm()
+                && !test_nasm_command()
+                && Some(true) != allow_prebuilt_nasm()
+            {
                 eprintln!(
-                    "Consider setting `AWS_LC_SYS_NO_ASM` in the environment for development builds.\
-                See User Guide about the limitations: https://aws.github.io/aws-lc-rs/index.html"
+                    "Consider setting `AWS_LC_SYS_PREBUILT_NASM` in the build environment.\
+                See User Guide: https://aws.github.io/aws-lc-rs/index.html"
```

Comment:
Deep link to the windows building page?

Reply:
I thought about that, but there's some refactoring I want to do to the User Guide so I'm hesitant to deep link.  In this case, the most relevant information might actually be on the "Resources" page where we list the environment variables that can affect the build: https://aws.github.io/aws-lc-rs/resources.html